xarray/core/datatree_mapping.py

```diff
@@ -68,6 +82,8 @@ def map_over_datasets(
     *args : tuple, optional
         Positional arguments passed on to `func`. Any DataTree arguments will be
         converted to Dataset objects via `.dataset`.
+    kwargs : dict, optional
+        Optional keyword arguments passed directly on to call ``func``.
```

Comment:
```suggestion
        Optional keyword arguments passed directly to ``func``.
```

Reply:
That's copied verbatim from apply_ufunc, but sure, looks a bit clunky.

Reply:
Of course also fine to keep as is for me.

Reply:
Maybe native speakers can chime in here, which formulation should be used here?

Reply:
I would use the latter.